build/helper/metadata_add_all.py

```diff
@@ -348,13 +348,15 @@ def add_all_function_metadata(functions, config):
 
 def _add_python_name(a, attributes):
     '''Adds 'python_name' - lower case + leading '_' if first character is a digit'''
```

Comment:
I think we should now assert if first character is digit.

Reply:
Done